smartcontractkit/external-adapters-js
# add por-indexer EA, add indexer to proof-of-reserves EA

## Description

Adds a `por-indexer` EA that connects to the new PoR indexer service, taking address sets and returning the sum of balances. Right now there is only a PoR indexer for Bitcoin, but this EA is written to allow any arbitrary PoR indexer service to be spun up and connected to.

## Changes

- New `por-indexer` EA
- `proof-of-reserves` EA updated to support this new `indexer`
- Request payload limit increased to `1MB` to support the big payloads



## Steps to Test

1. Spin up or get access to a `bitcoin-por-indexer` service.
2. Set `BITCOIN_MAINNET_POR_INDEXER_URL` to point to the above service.
3. Spin up `proof-of-reserves`, `por-indexer`, `wbtc-address-set` (and/or `renvm-address-set`) EAs
4. Run `test por-indexer` in monorepo root to run e2e tests.
5. Verify manually with curl:
  ```sh
  > curl \
		  -H "Content-Type: application/json" \
		  -d '{"id":"2","data":{"protocol": "wbtc", "indexer": "por_indexer"}}' \
		  http://proof_of_reserves_endpoint_here

  > {"jobRunID":"1","result":"24908724852868","statusCode":200,"data":{"result":"24908724852868"}}
  ```

## Quality Assurance
